GettingStarted/src/Samples/LocationAndRealTime/SGSLocationAndRealtimeVC.m

```diff
@@ -355,34 +319,30 @@ - (void)realTimeManager:(id<SITRealTimeInterface>)realTimeManager
 - (void)realTimeManager:(id<SITRealTimeInterface>)realTimeManager
  didUpdateUserLocations:(SITRealTimeData *)realTimeData
 {
-    [self displayRealtimeData:realTimeData];
+    [self showRealtimeMarkers:realTimeData];
 }
 
-#pragma mark - SITLocationDelegate Methods
-
-- (void)locationManager:(id<SITLocationInterface>)locationManager
-         didUpdateState:(SITLocationState)state
-{
-    NSLog(@"location manager changed to state: %d", state);
-    if (state == kSITLocationUserNotInBuilding) {
-        NSLog(@"Unable to find user on building. Please verify this building has been configured (calibrated) correctly");
-    }
+#pragma mark - Actions
+- (IBAction)reloadButtonPressed:(id)sender {
+    [self.errorLabel setHidden:YES];
+    [self.reloadButton setHidden:YES];
     
 }
 
-- (void)locationManager:(id<SITLocationInterface>)locationManager
-       didFailWithError:(NSError *)error
-{
-    NSLog(@"an error happened: %@", error);
+- (IBAction)locationButtonPressed:(id)sender {
+    if (!self.ready) {
+        return; // Wait until information is loaded
+    }
+    
+    [self changeLocationStatus];
 }
 
-- (void)locationManager:(id<SITLocationInterface>)locationManager
-      didUpdateLocation:(SITLocation *)location
-{
-    NSLog(@"new location update available: :%@", location);
-    [self updateLocation:location];
-    
+- (IBAction)realtimeButtonPressed:(id)sender {
+    if (!self.ready) {
+        return; // Wait until information is loaded
```

Comment:
Aqui non estaria ben avisar dalgun xeito porque se ignora o boton?

Reply:
Realmente sí, pero xa estaba así e era por pechar a tarefa. Ese exemplo vai haber que tocalo máis tarde